.github/renovate.json5

```diff
@@ -282,6 +282,18 @@
         "v1.13"
       ]
     },
+    {
+      // Do not upgrade etcd as they have removed bash starting from v3.5.5
+      "enabled": false,
+      "matchDepNames": [
+        "quay.io/coreos/etcd"
+      ],
+      "allowedVersions": "<3.5.5",
```

Comment:
Sorry, I just noticed now. `allowedVersions` looks superfluous due to `"enabled": false` (although harmless).

Reply:
ohh nice catch.